nodes/widgets/ui_control.js

```diff
@@ -177,13 +177,15 @@ module.exports = function (RED) {
                         // this message was sent by this particular node
                         if (evt === 'change') {
                             const wNode = RED.nodes.getNode(node.id)
-                            let msg = {
-                                payload: 'change',
-                                tab: payload.page, // index of tab
-                                name: payload.name // page name
+                            if (wNode && payload && wNode.send) {
+                                let msg = {
+                                    payload: 'change',
+                                    tab: payload.page, // index of tab
+                                    name: payload.name // page name
+                                }
+                                msg = addConnectionCredentials(RED, msg, conn, ui)
+                                wNode.send(msg)
```

Comment:
This guarding has slipped in Joe.

Happy to remove it

FWIW, I had several issues over the course of developing the wysiwyg and ui-spacer node where by the dashboard became unavailable due to an undefined thing in the event handler.